manifests/init.pp

```diff
@@ -1,7 +1,7 @@
 class dhcp (
   $dnsdomain,
   $nameservers,
-  $ntpservers          = undef,
+  $ntpservers          = [],
```

Comment:
This should come with updates in `https://github.com/puppetlabs/puppetlabs-dhcp/blob/master/spec/classes/dhcp_spec.rb` to make sure `$ntpservers` isn't an empty array